### PR TITLE
Style specialties parent page

### PR DESCRIPTION
**Pull request by hannakim91**
# To Do/Questions:

## Images:
- [x] Need art for the 2 first ones under “high impact, low ego” and  the one next to “our fleet of specialties”
- [x] Cloud native SVG missing
- [ ] What image to use for blogs? Maybe starting now we can all add a corresponding image? or author image?

## Links:
- [x] Specialty case studies for “see it in action” - should these buttons be removed until there are relevant case studies? Also which case studies are preferred for ones with multiple (like ember)?
- [ ] Where is “read on” button supposed to link to?
- [ ] "Future in fintech" url? 

## Accessibility/SEO 
(Maybe on separate ticket since these points affect multiple pages?)

- [x] *Critical:* Add alt tags after all pages have proper images
~- [ ] *Serious:* Contrast issues with background color & text - lighthouse and axe dev tools both show there isn't enough contrast for: the white text on red background, light grey text on white bg, white text on navy bg (see [here](https://www.essentialaccessibility.com/blog/accessibility-for-people-with-astigmatism#:~:text=There%20is%20a%20myth%20about,with%20astigmatism%20called%20%E2%80%9Chalation%E2%80%9D.) and some of the comments [here](https://uxmovement.com/buttons/the-myths-of-color-contrast-accessibility/) say white text is hard to read for some people with low vision issues. Is this something to ask the designer for input on?~ future update
- [x] *Moderate:* H-tags in proper order (use `H2` instead of `H3` for secondary headers, e.g. "high impact, low ego," and style size properly; same for `SingleQuote` name/title and `WaveFooter` services
- [x] Document does not have a valid rel=canonicalPoints to a different domain (https://shipshape.io/services/)
- [ ] Links do not have descriptive text ("learn more" buttons for specialties children pages)
- [ ] Tap targets are not sized appropriately - `WaveFooter` buttons need more space for mobile friendliness - overlapping with each other

_*Ratings coming from axe dev tools_

## Etc:
- [x] Divide-gray-light // divide-gray-### not working 
- [x] Latest podcast data (title/date/blurb) to populate that section, like with latest blog

### CODE REVIEW

File: pages/services/index.vue [hunk 1]
```diff
@@ -1,15 +1,12 @@
 <template>
-  <div>
-    <HeroBlock hero-classes="services">
+  <div itemscope itemtype="http://schema.org/Service">
```
Comment (rwwagner90) [inline pages/services/index.vue:2]: I don't think we want this `itemscope itemtype` stuff for this page. It's generally for a specific service, and provides the metadata to tell google about that service.

File: pages/services/index.vue [hunk 2]
```diff
@@ -19,6 +19,7 @@
           format="webp"
           src="/img/hero-images/services.png"
           width="1344"
+          alt=""
```
Comment (rwwagner90) [inline pages/services/index.vue:22]: @hannakim91 do you plan to come back through and fill these in later? I think we should definitely make our alt tags descriptive.

Comment (hannakim91) [reply pages/services/index.vue:22]: @rwwagner90 The w3 docs say to leave a null alt tag when it's a decorative image: I feel like the hero images are more-so ambiance/eye-candy vs things that would need to be read for context by screen readers?https://www.w3.org/WAI/tutorials/images/decorative/#image-used-for-ambiance-eye-candy

Comment (hannakim91) [reply pages/services/index.vue:22]: Also thinking another run-through with manual tests like being able to tab through the pages w/o getting stuck in a section would be good - making a to-do list of those.

Comment (rwwagner90) [reply pages/services/index.vue:22]: > The w3 docs say to leave a null alt tag when it's a decorative image: I feel like the hero images are more-so ambiance/eye-candy vs things that would need to be read for context by screen readers?https://www.w3.org/WAI/tutorials/images/decorative/

I'm not sure I agree with that. Aren't all images decorative in that sense? I tend to provide descriptive tags for all of them. For example `alt="Ship Shape octopus mascot drinking a coffee and using a laptop."`. My thought was for every image a sighted person can see, there should be meaningful content for those using screen readers, but I could definitely be wrong on my understanding of that if those descriptions are actually annoying to people with screen readers.

Comment (hannakim91) [reply pages/services/index.vue:22]: That was my understanding of it - like the alt text for the top hero image would be something like `Ship Shape octopus mascot holds a sign in an ocean filled with browser windows` doesn't really provide additional context for users with vision issues, and may be more confusing in the context of the header/CTA button?

In comparison to these (informative image examples)[https://www.w3.org/WAI/tutorials/images/informative/] - which have a reason for users to know what the images contain.

(This person)[https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm] talks about similar use cases with stock images for "flavoring" blog posts too.

File: pages/services/index.vue [hunk 3]
```diff
@@ -22,21 +19,306 @@
...
+              class="h-auto m-auto max-w-2xl w-full lg:order-last"
+              format="webp"
+              height="1034"
+              src="/img/services/build.png"
+              width="1312"
+              alt=""
+            />
+            <p>
+              Our software engineers specialize in
+              <nuxt-link to="/services/cloud-native-consulting/"
```
Comment (rwwagner90) [inline pages/services/index.vue:79]:
```suggestion
              <nuxt-link to="/services/ember-consulting/"
```

File: pages/services/index.vue [hunk 4]
```diff
@@ -22,21 +19,306 @@
...
+              class="h-auto m-auto max-w-2xl w-full"
+              format="webp"
+              height="950"
+              src="/img/hero-images/open-source.png"
+              width="1312"
+              alt=""
+            />
+            <p>
+              From the future of fintech, to
+              <nuxt-link to="/work/enigma">cybersecurity</nuxt-link>, and more,
```
Comment (rwwagner90) [inline pages/services/index.vue:119]:
```suggestion
              <nuxt-link to="/work/expel">cybersecurity</nuxt-link>, and more,
```

File: tailwind.config.js [hunk 5]
```diff
@@ -95,8 +95,10 @@ module.exports = {
       20: '5rem',
       24: '6rem',
       32: '8rem',
+      36: '9rem',
```
Comment (rwwagner90) [inline tailwind.config.js:98]: I'm curious, what were these additional sizes needed for?

**Review by rwwagner90 — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 5 above)
Comment (hannakim91) [reply tailwind.config.js:98]: Mostly used them for padding/margins in some areas where the sizes seemed to match the spec better than jumping 2 rem

**Pull request closed without merging**